vapp/src/veCurveVault.vue

```diff
@@ -70,7 +70,7 @@ export default {
       this.drizzleInstance.contracts['veCurveVault'].methods['depositAll'].cacheSend({from: this.activeAccount})
     },
     on_zap() {
-      this.drizzleInstance.contracts['CurveVestingBackscratch'].methods['zap'].cacheSend({from: this.activeAccount})
+      this.drizzleInstance.contracts['CurveBackzapper'].methods['zap'].cacheSend({from: this.activeAccount})
```

Comment:
This will need to be updated to actually find which gauges to use and pass those in as a param